### PR TITLE
Update universal CSS for legacy IE browsers (Fixes #7390)

### LINKED ISSUE #7390: Improve universal stylesheet served to old IE browsers
## Description

The universal stylesheet we serve to IE8 and below could benefit from some minor tweaks to make it a bit more readable:

Screenshot 2019-07-08 at 14 11 07
 
1. The navigation contains many links, which pushes the main page content down a long way. We could provide a really basic layout for this so that it doesn't take up so much height.
2. Hide the useless `<button>` elements (e.g. "Menu", "Close Firefox Menu"), that don't do anything without the JS.
3. Improve readability by placing all content within a centered container in the page and a set max-width.
4. Hide content that is only really meant for Firefox users.
5. Anything else?

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Browsing the no-frills site is a slightly more pleasant experience.

#### Tasks
- [x] Update legacy IE stylesheets.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Updates universal stylesheet served to legacy IE browsers.
- Tested in IE 6, 7 & 8.

## Issue / Bugzilla link
#7390

## Testing
- [x] Protocol pages should be more readable.
- [x] Pebbles pages should be more readable.
- [x] Sandstone pages should be more readable.

### CODE REVIEW

**Comment by alexgibson** [comment]
Screenshots:

**Homepage before & after:**

homepage-ie

**/firefox before & after**

firefox-ie

**/accounts before & after**

accounts-ie

**/channel before & after**

channel-ie

**Comment by stephaniehobson** [comment]
Vast improvement from a menu perspective 💯 And the new `.is-js-dependent` class is 🔥

File: bedrock/firefox/templates/firefox/accounts-2019.html [hunk 1]
```diff
@@ -56,7 +56,7 @@ <h1 class="c-accounts-hero-title">{{ _('There is a way to protect your privacy.
     </div>
 
     {% if has_signed_in_copy %}
-    <div class="c-accounts-hero-body c-accounts-hero-body-signed-in">
+    <div class="c-accounts-hero-body c-accounts-hero-body-signed-in is-js-dependent">
```
[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/firefox/templates/firefox/accounts-2019.html]: Old IE doesn’t load the CSS or JS that gets used to hide/show this piece of content, hence why the class here is needed.

**Comment by alexgibson** [comment]
Thanks for the review @stephaniehobson, merging!

**Pull request merged**